packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -112,6 +113,7 @@ const GridTile: FC<
                 <MetricQueryDataProvider
                     metricQuery={savedQuery?.metricQuery}
                     tableName={savedQuery?.tableName || ''}
+                    appliedDashboardFilters={dashboardFilters}
```

Comment:
since we changed to `useSavedQueryWithDashboardFilters` I don't think we need `appliedDashboardFilters` has the filters should already be port of the `metricQuery` param.  Might be the reason why you get the duplicate filter.

Reply:
done! updated the description with a new screen recording 👍